### ISSUE TITLE
v0.0.12 doesn't work anymore

### ISSUE THREAD

**Issue by raykin** [capacitor-community/apple-sign-in#47]
**Describe the bug**
 v0.0.12 can't work anymore because package.json contains dependency to latest Capacitor which points to latest released Cap3.

**To Reproduce**
Install v0.0.12 in a fresh new app, ionic serve report compile error.
```
[ng] Error: node_modules/@capacitor-community/apple-sign-in/node_modules/@capacitor/core/types/definitions-internal.d.ts:16:18 - error TS2430: Interface 'CapacitorInstance' incorrectly extends interface 'CapacitorGlobal'.
[ng]   Types of property 'Plugins' are incompatible.
[ng]     Property 'SignInWithApple' is missing in type '{ [pluginName: string]: { [prop: string]: any; }; }' but required in type 'PluginRegistry'.
[ng] 16 export interface CapacitorInstance extends CapacitorGlobal {
[ng]                     ~~~~~~~~~~~~~~~~~
[ng]   node_modules/@capacitor-community/apple-sign-in/dist/esm/definitions.d.ts:3:9
[ng]     3         SignInWithApple: SignInWithApplePlugin;
[ng]               ~~~~~~~~~~~~~~~
[ng]     'SignInWithApple' is declared here.
```

Create a new branch to support cap2 is possible. I tried to sent a pull request but it can't work because v0.0.12 is a tag. 

![Screen Shot 2021-08-06 at 4 01 54 PM](https://user-images.githubusercontent.com/490502/128478642-6084dec1-ab98-4563-880f-4547c7176c35.png)

**Comment by selected-pixel-jameson** [comment]
The same things happens with Capacitor 3 on the latest version 1.0.0.

**Comment by jcesarmobile** [comment]
I've published 0.2.0 that should be compatible with Capacitor 2, and 1.0.1 that should be compatible with Capacitor 3

**Comment by raykin** [comment]
It doesn't work. Becase 0.2.0 is now using Capacitor3 compatible code, although the dependency has fixed.
![Screen Shot 2021-09-20 at 10 21 59 PM](https://user-images.githubusercontent.com/490502/134019648-95cf9ec6-7b91-47ba-9341-da91d9a77650.png)
